### PR TITLE
feat: add UiBulletPointsItem small variant

### PR DESCRIPTION
**Pull request by dolaron**
We would like to have a small variant of UiBulletPointItem

### CODE REVIEW

**Review by AlanLes — DISMISSED** [review]
LGTM! Awesome job! Makes sense! 🎉

File: src/components/molecules/UiBulletPoints/_internal/UiBulletPointsItem.vue [hunk 1]
```diff
@@ -110,6 +110,14 @@ const isUnordered = computed(() => tag.value === 'ul');
   justify-content: functions.var($element, justify-content, flex-start);
   gap: functions.var($element, gap, var(--space-12));
 
+  &--small {
+    font: functions.var($element, font, var(--font-body-2-comfortable));
```
Comment (pspaczek) [inline src/components/molecules/UiBulletPoints/_internal/UiBulletPointsItem.vue:114]: like everyone you miss set `letter-spacing` :(

**Review by pspaczek — CHANGES_REQUESTED** [review]

File: src/components/molecules/UiBulletPoints/_internal/UiBulletPointsItem.vue [hunk 2]
```diff
@@ -110,6 +110,14 @@ const isUnordered = computed(() => tag.value === 'ul');
...
+
+    .ui-text {
```
Comment (pspaczek) [inline src/components/molecules/UiBulletPoints/_internal/UiBulletPointsItem.vue:116]: you can't use classes to do that.

File: src/components/molecules/UiBulletPoints/_internal/UiBulletPointsItem.vue [hunk 3]
```diff
@@ -110,6 +110,14 @@ const isUnordered = computed(() => tag.value === 'ul');
   justify-content: functions.var($element, justify-content, flex-start);
   gap: functions.var($element, gap, var(--space-12));
 
+  &--small {
```
Comment (pspaczek) [inline src/components/molecules/UiBulletPoints/_internal/UiBulletPointsItem.vue:113]: modifiers should be after elements

**Comment by pspaczek** [comment]
@dolaron this PR doesn't make sens for me. Because you can use 
```
 <template
        v-for="(item, key) in items"
        :key="key"
      >
        <UiBulletPointsItem
          :icon="item.icon"
          :text-content-attrs="{
            class:"ui-text--body-2-comfortable",
          }"
        >
          {{ item }}
        </UiBulletPointsItem>
      </template>
```

**Comment by pspaczek** [comment]
@dolaron IMO it should be 
* you miss set a new gap 🙈 
```
.ui-bullet-points {
...
  &--small {
    --bullet-points-gap: var(--space-8);
    --text-font: var(--font-body-2-comfortable);
    --text-letter-spacing: var(--letter-spacing-body-2-comfortable);
  }
}
</style>
```
and in code you add only `class="ui-bullet-points--theme-secondary ui-bullet-points--small"` to make this 
Zrzut ekranu 2023-09-20 o 13 28 44

**Pull request merged**